pkgs/applications/networking/browsers/brave/default.nix

```diff
@@ -65,32 +66,56 @@ let rpath = lib.makeLibraryPath [
     udev
     xorg.libxcb
     zlib
+    xdg_utils
 ];
 
 
 in stdenv.mkDerivation rec {
     name = "brave";
-    version = "0.25.2";
+    version = "0.55.22";
 
     src = fetchurl {
-        url = "https://github.com/brave/browser-laptop/releases/download/v${version}dev/brave_${version}_amd64.deb";
-        sha256 = "1r3rsa6szps7mvvpqyw0mg16zn36x451dxq4nmn2l5ds5cp1f017";
+        url = "https://github.com/brave/brave-browser/releases/download/v${version}/brave-browser_${version}_amd64.deb";
+        sha256 = "0111wbzyp2n8s0yskzq7xp7ipv5l6nyqzjczhh23j8yagm084kki";
     };
 
     phases = [ "unpackPhase" "installPhase" ];
 
     nativeBuildInputs = [ dpkg ];
 
-    unpackPhase = "dpkg-deb -x $src .";
+    unpackPhase = "dpkg-deb --fsys-tarfile $src | tar -x --no-same-permissions --no-same-owner";
 
     installPhase = ''
         mkdir -p $out
 
         cp -R usr/* $out
+        cp -R opt/ $out/opt
+
+        export BINARY=$out/opt/brave.com/brave/brave-browser
+
+        # Fix path to bash in $BINARY
+        substituteInPlace $BINARY \
+            --replace /bin/bash ${stdenv.shell}
+
+        ln -sf $BINARY $out/bin/brave-browser
 
         patchelf \
             --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-            --set-rpath "${rpath}" $out/bin/brave
+            --set-rpath "${rpath}" $BINARY
+
+        # Fix paths
+        substituteAllInPlace $out/applications/brave-browser.desktop \
```

Comment:
Shouldn't this still be `$out/share/application/brave-browser.desktop`

Reply:
Fixed.